src/entities/filter.ts

```diff
@@ -16,10 +16,10 @@ export interface Filter {
   phrase: string;
   /** The contexts in which the filter should be applied. */
   context: FilterContext[];
-  /** When the filter should no longer be applied */
-  expiresAt?: string | null;
   /** Should matching entities in home and notifications be dropped by the server? */
   irreversible: boolean;
   /** Should the filter consider word boundaries? */
-  wholeWord: string;
+  whole_word: boolean;
+  /** When the filter should no longer be applied */
+  expires_at?: string | null;
```

Comment:
See https://github.com/neet/masto.js/issues/725#issuecomment-1334107207

Reply:
Applied the requested change!